aspnet/whitepapers/aspnet-and-iis6.md

```diff
@@ -29,39 +29,39 @@ To start the wizard - note, to run the wizard you must be logged in as an admini
 
 Once selected you should see the 'Configure Your Server Wizard' opening screen:
 
-![](aspnet-and-iis6/_static/image1.jpg)
+![Screenshot of the Windows configure your server wizard. The next button is highlighted.](aspnet-and-iis6/_static/image1.jpg)
 
 Click 'Next &gt;':
 
-![](aspnet-and-iis6/_static/image2.jpg)
+![Screenshot of the Windows preliminary steps screen of the configure your server wizard. The next button is highlighted.](aspnet-and-iis6/_static/image2.jpg)
 
 Click 'Next &gt;'
 
-![](aspnet-and-iis6/_static/image3.jpg)
+![Screenshot of the Windows configure your server wizard screen titled server role. The menu shows the application server option highlighted.](aspnet-and-iis6/_static/image3.jpg)
 
 On this screen you will need to select 'Application server (IIS, ASP.NET) as the options to configure.
 
 Click 'Next &gt;'.
 
-![](aspnet-and-iis6/_static/image4.jpg)
+![Screenshot of the  Windows configure your server wizard screen titled application server options. The Enable A S P dot net box is checked.](aspnet-and-iis6/_static/image4.jpg)
 
 After selecting to configure the server as an Application Server, this screen will be displayed prompting what additional capabilities should be installed. Neither option is selected by default. To enable ASP.NET automatically, you need to select 'Enable ASP.NET'.
 
 Click 'Next &gt;'.
 
-![](aspnet-and-iis6/_static/image5.jpg)
+![Screenshot of the Windows configure your server wizard screen titled summary of selections. The next button is highlighted.](aspnet-and-iis6/_static/image5.jpg)
 
 This screen displays what options are to be installed.
 
 Click 'Next &gt;'.
 
-![](aspnet-and-iis6/_static/image6.jpg)
+![Screenshot of the Windows configure your server wizard screen titled applying selections. A status bar is shown partially completed.](aspnet-and-iis6/_static/image6.jpg)
```

Comment:
Suggestion: Screenshot of the configure your server wizard screen. The applying selections status bar displays.